app/lib/registration/username_onboarding_screen.dart

```diff
@@ -21,6 +23,8 @@ class UsernameOnboardingScreen extends HookWidget {
   @override
   Widget build(BuildContext context) {
     final loc = AppLocalizations.of(context);
+    final colors = CustomColorScheme.of(context);
+    final Color backgroundColor = colors.backgroundBase.secondary;
     final registrationState = context.watch<RegistrationCubit>().state;
```

Comment:
This state is actually only needed for the initial value of the text field. Also it does not change. So, we could just use `initialValue` of `TextFormField` in `_UsernameTextField`. This also removes unneeded text controller. The submit function can get the username value via an argument.